squirrel/serialization/msgpack.py

```diff
@@ -56,7 +56,11 @@ def serialize_shard_to_file(
 
     @staticmethod
     def deserialize_shard_from_file(
-        fp: str, fs: t.Optional[AbstractFileSystem] = None, mode: str = "rb", **open_kwargs
+        fp: str,
+        fs: t.Optional[AbstractFileSystem] = None,
+        mode: str = "rb",
+        unpacker_kwarg: t.Optional[t.Dict] = None,
```

Comment:
How about an `s` here
```suggestion
        unpacker_kwargs: t.Optional[t.Dict] = None,
```